requirements.in

```diff
@@ -30,7 +30,7 @@ python-dotenv==0.20.0
 # PaaS
 awscli-cwlogs>=1.4,<1.5
 itsdangerous==2.1.2
-notifications-utils @ git+https://github.com/alphagov/notifications-utils.git@56.0.0
+notifications-utils @ git+https://github.com/GSA/notifications-utils.git
 govuk-frontend-jinja @ git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.5.8-alpha
```

Comment:
Before pilot, should append @commithash to fix the lib's version

[See issue #38](https://github.com/GSA/notifications-admin/issues/38)